### PR TITLE
Add basic bootstrap AUC and coxph to cohorts

### PR DESCRIPTION
**Pull request by tavinathanson**
I keep using these and figured they should live in `cohorts`.

@arahuja @jburos

### CODE REVIEW

File: cohorts/model.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    pred_col : the column we're trying to predict
+    n_boostrap : the number of bootstrap samples
+
+    Returns
+    -------
+    list : AUCs for each sampling
+    """
+    scores = np.zeros(n_bootstrap)
+    count_col, df = cohort.as_dataframe(func)
+    preds = df[pred_col].astype(int)
```
Comment (arahuja) [inline cohorts/model.py:52]: similar code exists here: https://github.com/hammerlab/cohorts/blob/master/cohorts/plot.py#L152-L159, do you want to refactor the plotting coding to use this as well?

File: cohorts/model.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        if is_single_class(sampled_pred, col=pred_col):
+            continue
+        scores[i] = roc_auc_score(sampled_pred, sampled_counts)
+    return scores
+
+def mean_bootstrap_auc(cohort, func, pred_col="is_benefit", n_bootstrap=1000):
+    return bootstrap_auc(cohort, func, pred_col, n_bootstrap).mean()
+
+def coxph_model(formula, data, time_col, event_col, **kwargs):
+    sdata = patsy.dmatrix(
```
Comment (arahuja) [inline cohorts/model.py:73]: add patsy to `setup.py`?

Comment (tavinathanson) [inline cohorts/model.py:73]: Yikes, had a note to do that and forgot!

File: cohorts/model.py (hunk 1 above)
Comment (tavinathanson) [inline cohorts/model.py:52]: Good call, I should.

File: cohorts/model.py (hunk 2 above)
Comment (tavinathanson) [inline cohorts/model.py:73]: Done

File: cohorts/model.py (hunk 1 above)
Comment (tavinathanson) [inline cohorts/model.py:52]: Done

**Comment by tavinathanson** [comment]
@arahuja just addressed your comments, good to merge?

**Comment by arahuja** [comment]
👍

**Pull request merged**